### PR TITLE
Add Bio for bgirgi

### PR DESCRIPTION
**Pull request by bgirgi**
fixes Issue #17 by creating a new bio file for bgirgi

### CODE REVIEW

File: bgirgi.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# GitHub Student
+
+* Name: bgirgi
```
Comment (mlmallo) [inline bgirgi.md:3]: Looks good! Where are you located?

File: bgirgi.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# GitHub Student
+
```
Comment (bgirgi) [inline bgirgi.md:2]: Maybe this should have your city?

File: bgirgi.md (hunk 1 above)
Comment (msouna) [inline bgirgi.md:3]: Can you add your location ?

**Comment by mlmallo** [comment]
@bgirgi - awesome!  I have never visited Montreal. :+1:

**Pull request merged**